PythonDangerfile

```diff
@@ -4,7 +4,7 @@ danger.import_dangerfile(github: "loadsmart/dangerfile", :path => "Dangerfile")
 fail("(i)pdb left in the code") if `find . -iname '*.py' | xargs grep -r "import [i]pdb"`.length > 1
 
 # Check if diff contains 'import *'
-python_files = (git.modified_files + git.added_files).select { |file| file.end_with? ".py" }
+python_files = (git.modified_files + git.added_files).select { |file| file.end_with? ".py" and not file.include? "=>" }
```

Comment:
I would do this to be clearer, but it is up to you:

python_files = (git.modified_files + git.added_files).select { |file| file.end_with? ".py"}.reject { |file| file.include? "=>" }

or something like

python_files = (git.modified_files + git.added_files).select { |file| file.end_with? ".py"}
python_files = python_files.reject { |file| file.include? "=>" }

Reply:
Done. Thanks for the advice.